Multisig/Logic/Models/Safe.swift

```diff
@@ -78,15 +78,19 @@ extension Safe: Identifiable {
     static func edit(address: String, name: String) {
         dispatchPrecondition(condition: .onQueue(.main))
         let context = App.shared.coreDataStack.viewContext
-
         let fr = Safe.fetchRequest().by(address: address)
-        
         guard let safe = try? context.fetch(fr).first else { return }
-        
         safe.name = name
-
         App.shared.coreDataStack.saveContext()
     }
+
+    static func select(address: String) {
```

Comment:
Do we need this?, we used to have Selection.select right?

Reply:
no, there's no Selection.select(address), so I've added this static method when I don't have the Safe object